app/routes.py

```diff
@@ -1,2 +1,237 @@
-from flask import Blueprint
+import requests
+from flask import Blueprint, jsonify, make_response, request
+from app import db
+from app.models.customer import Customer
+from app.models.rental import Rental
+from app.models.video import Video
 
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+# START OF CUSTOMER CRUD ROUTES
+
+
+@customers_bp.route("", methods=["GET"])
+def list_all_customers():
+    customers_response = [customer.as_dict() for customer in Customer.query.all()]
+    return jsonify(customers_response)
+
+
+@customers_bp.route("", methods=["POST"])
+def create_customer():
+    request_body = request.get_json()
+
+    if invalid_customer_post_request_body(request_body):
+        return make_response({"details": "Missing required data"}, 400)
+
+    customer = Customer(name=request_body["name"], postal_code=request_body["postal_code"], phone=request_body["phone"])
+
+    db.session.add(customer)
+    db.session.commit()
+
+    return make_response({"id": customer.customer_id}, 201)
+
+
+def invalid_customer_post_request_body(request_body):
```

Comment:
👍  one thing we could do is take any helper functions and make a separate file solely for them, so they aren't cluttering up `routes.py`